tsl/test/expected/cagg_joins-12.out

```diff
@@ -310,7 +310,8 @@ SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
 FROM conditions, devices_view
 WHERE conditions.device_id = devices_view.device_id
 GROUP BY name, bucket, devices_view.device_id;
-ERROR:  joins for hierarchical continuous aggregates are not supported
+NOTICE:  refreshing continuous aggregate "conditions_summary_daily_view"
```

Comment:
I see that these statements need the mentioned feature - but don't we need real queries/plan-s to track that it works as expected?

Reply:
I agree with @kgyrtkirk and wonder if we should add some JOIN tests in the caggs_on_caggs suite where we extensively test it checking plans, doing different time bucket data type tests... caggs on caggs validation tests... etc. WDYT @RafiaSabih ?